proposals/proposal_process.md

```diff
@@ -0,0 +1,183 @@
+affect the developer-visible surface).
+
+**Address all browser-specific considerations.**  Very likely, individual
+browsers will have separate design docs detailing the browser-specific aspects
+of the API.  These are out of scope of the WECG.  If appropriate, the author
+may include browser-specific considerations as addenda or useful background.
+
+**Replace a proper, detailed, web-style spec.**  While we do not currently have
+complete specs for extension APIs, we intend to (gradually) produce them.
+These API proposals should deal with higher-level discussion of the API as a
+whole, and should not include exact spec language.
+
+**Have API proposals be perennially "living" documents.**  An API proposal
+should be kept up-to-date during the implementation period of the initial API
+or change it proposes.  Once that initial version or change has been
+implemented, the proposal is complete.  Any changes should have a separate
+proposal.
+
+## Prior Art
+
+This proposal is heavily inspired by the existing
+[Chrome Extension API Proposal Process](https://chromium.googlesource.com/chromium/src/+/HEAD/extensions/docs/new_api_proposal.md)
+and [API Overview Document](https://docs.google.com/document/d/182XXEPwbh5dyMTO_Q3bZ9k4PYY19Woydu5b3XvPoSmc/edit#heading=h.x9snb54sjlu9)
+(which has been used for many new extension APIs and has been a jumping off
+point for some new proposals in the WECG).
+
+## When to Use this Proposal
+
+This proposal should be used for any significant new changes or additions to
+public extension APIs.
+
+## When to Not Use this Proposal
+
+Browsers do not need to use this proposal for any APIs that will never become
+public.  As an example, in Chrome, there is the concept of "private APIs" that
+are used by [component extensions](https://chromium.googlesource.com/chromium/src/+/main/extensions/docs/component_extensions.md)
+-- extensions that are built into the browser as an implementation detail.  In
+these cases, browser vendors do not need to (and should not!) put forth an API
+proposal in this forum.  (Though note that browsers may still have separate
+processes for such APIs.)
+
+## How to Use this Proposal
+
+*   Find (or represent) a browser willing to implement the API imminently and
+    propose the API.  We want to ensure these API proposals don't become a
+    collection of "aspirational" APIs -- things that we would ideally like to
+    get to at some point.  These proposals should have thorough (and thus,
+    potentially costly) reviews, beyond what we expect for a general overview
+    of a suggestion in a filed issue.  As such, these proposals should only be
+    used for APIs that have support and commitment to implement from at least
+    one browser.
+*   Fill out the API Proposal.  There are instructions in each section.  For any
+    section that is not relevant, you may put "N/A" (likely with a brief summary
+    of why the section is not relevant, if it is not immediately obvious).
```

Comment:
The template is very detailed with its instructions. Is the author supposed to delete the instructions make the doc more concise? If yes, let's emphasize that here (and maybe also in the template)